src/core/components/SlQueryBuilder/SlQueryBuilder.test.tsx

```diff
@@ -59,4 +59,39 @@ describe('SlQueryBuilder', () => {
 
     expect(queryBuilderFieldLabels).toEqual(expectedFieldLabels);
   });
+
+  it('should pass disabled prop to QueryBuilder', () => {
+    const queryBuilderSpy = jest.spyOn(QueryBuilderModule, "default").mockImplementation(jest.fn());
+    
+    const reactNode = React.createElement(SlQueryBuilder, { 
+      value: '', 
+      customOperations, 
+      fields, 
+      messages: {}, 
+      onChange: jest.fn(),
+      disabled: true 
+    });
+    render(reactNode);
+
+    const queryBuilderProps = queryBuilderSpy.mock.lastCall?.at(0);
+    expect(queryBuilderProps?.disabled).toBe(true);
+  });
```

Comment:
```suggestion
  it('should pass disabled prop to QueryBuilder', () => {
    const queryBuilderSpy = jest.spyOn(QueryBuilderModule, "default").mockImplementation(jest.fn());

    renderElement(customOperations, fields, disabled);

    const queryBuilderProps = queryBuilderSpy.mock.lastCall?.at(0);
    expect(queryBuilderProps?.disabled).toBe(true);
  });
```

Let's update the `renderElement` to accept a new optional argument `disabled`.